### DISCUSSION TITLE
geomap: allow geojson data coming from a rest api or db etc

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#51347] [Dashboards, Panels & Visualizations]
Greetings!

Would be nice if geomap allowed geojson coming not just from maps/files or URL but also from another data source  query, just as you can do it for lon/lat

Right now I can do the following by using my own rest api

<img width="231" alt="image" src="https://user-images.githubusercontent.com/900951/175378833-624f17eb-873c-4263-8cf2-de79aed93bce.png">

But I want to do this from another data source and plugin the geojson column returned to plug it in as a value

![image](https://user-images.githubusercontent.com/900951/175379344-84cc32fc-549c-4004-8adf-6c453399afb2.png)

Just as I can enter/select long lat from data source query

<img width="200" alt="image" src="https://user-images.githubusercontent.com/900951/175379593-1cd9acf1-cd4e-42b8-84cd-961525c490be.png">

Thanks for lovely/powerful grafana

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
